3542/3/db/seeds.rb

```diff
@@ -0,0 +1,18 @@
+require 'active_record'
+require 'csv'
+Dir['./models/*.rb'].each { |file| require file }
+
+ActiveRecord::Base.establish_connection(adapter: 'sqlite3',
+                                        database: 'db/development.sqlite3',
+                                        pool: 15,
+                                        timeout: 5000)
+
+NAME = 0
+ADDRESS = 1
+DESCRIPTION = 2
+
+CSV.foreach(File.realpath('db/data/data.csv')) do |row|
+  Place.find_or_create_by(name: row[NAME],
```

Comment:
when I run seeds changing only description, I get duplicates (multiple places with the same name).
I want to identify place by name.
Change seeds so that it find_or_create by name only 
and then update address and description